shadow-cljs.edn

```diff
@@ -2,7 +2,7 @@
 {:source-paths ["src" "test/cljs"]
 
  :dependencies [[reagent "1.0.0"]
-                [re-frame "0.12.0"]
+                [re-frame "1.3.0"]
```

Comment:
Wondering if cider-nrepl, piggieback versions should be updated in shadow-cljs.edn as well...?

Reply:
We might not need piggieback anymore: [Does shadow-cljs use Piggieback? No, it doesn't.](https://github.com/nrepl/piggieback#does-shadow-cljs-use-piggieback)

Reply:
> Wondering if cider-nrepl, piggieback versions should be updated in shadow-cljs.edn as well...?
> 
> ...or maybe removed, since I think we're all using local updated versions anyways...?

I would prefer if other contributors could share their opinion about that. I would certainly remove cider-nrepl and piggieback, but I'm not sure if it could affect others.

It's worth creating an issue anyway. Thanks for mentioning @erikseppanen

Reply:
Removing cider-nrepl might hurt emacs users. Cider needs the nrepl dep for the editor to jack in.

Reply:
@shivekkhurana, I think what @erikseppanen was hinting at is that editor specific deps could be removed from the repo since each developer can use their own preferred versions of these libs, and the ones in the repo are quite outdated anyways. For example, by overwriting deps in `~/.shadow-cljs/config.edn`

But for now I'm going to keep these libs in the repo :)

Reply:
Yes thank you @ilmotta, that's what I was referring to. There's a [small reference](https://github.com/status-im/status-mobile/blob/develop/doc/ide-setup.md#using-emacscider) to it in the docs.

I can see some value in maintaining them in `shadow-cljs.edn` so that a dev wouldn't *have* to maintain a separate `~/.shadow-cljs/config.edn`. But IIRC, the emacs cider packages (for each developer's local emacs installation) need to be correspondingly up-to-date, or you get a warning about packages being out-of-sync.

In general, when I see those outdated deps, I wonder to whose benefit they are there. I guess it doesn't matter that much leaving them in there... it's a low priority.